packages/compass-aggregations/src/modules/pipeline-builder/pipeline-ai.ts

```diff
@@ -34,18 +37,23 @@ export const initialState: AIPipelineState = {
   errorMessage: undefined,
   isInputVisible: false,
   aiPipelineFetchId: -1,
+  isGuideCueVisible: false,
+  guideCueTitle: undefined,
+  guideCueDescription: undefined,
 };
 
 export const enum AIPipelineActionTypes {
   AIPipelineStarted = 'compass-aggregations/pipeline-builder/pipeline-ai/AIPipelineStarted',
   AIPipelineCancelled = 'compass-aggregations/pipeline-builder/pipeline-ai/AIPipelineCancelled',
   AIPipelineFailed = 'compass-aggregations/pipeline-builder/pipeline-ai/AIPipelineFailed',
   AIPipelineSucceeded = 'compass-aggregations/pipeline-builder/pipeline-ai/AIPipelineSucceeded',
+  AIPipelineCreatedFromQuery = 'compass-aggregations/pipeline-builder/pipeline-ai/AIPipelineCreatedFromQuery',
   CancelAIPipelineGeneration = 'compass-aggregations/pipeline-builder/pipeline-ai/CancelAIPipelineGeneration',
+  HideGuideCue = 'compass-aggregations/pipeline-builder/pipeline-ai/HideGuideCue',
   ShowInput = 'compass-aggregations/pipeline-builder/pipeline-ai/ShowInput',
   HideInput = 'compass-aggregations/pipeline-builder/pipeline-ai/HideInput',
   ChangeAIPromptText = 'compass-aggregations/pipeline-builder/pipeline-ai/ChangeAIPromptText',
-  LoadGeneratedPipeline = 'compass-aggregations/LoadGeneratedPipeline',
+  LoadAIPipeline = 'compass-aggregations/LoadAIPipeline',
```

Comment:
While makes it definitely clearer, we currently have 3 people working on the code in the same area, I'd maybe stay away from renaming for now just to limit the amount of possible git conflicts between PRs